src/services/dispatchers/dispatchers/types.ts

```diff
@@ -19,6 +20,31 @@ export interface DispatchersService {
    */
   find(options?: OptionsWithRql): Promise<PagedResult<Dispatcher>>;
 
+  /**
+   * ## Retrieve a list of all Dispatchers
+   *
+   * **Global Permissions:**
+   * - `VIEW_DISPATCHERS` - Allows a user to view Dispatchers
+   *
+   * @param options {@link OptionsWithRql} - Add filters to the requested list
+   * @returns An array of Dispatchers {@link Dispatcher Dispatcher[]}
+   * @throws {@link Error} Do not pass in limit operator with findAll
+   * @throws {@link NoPermissionError} when the user doesn't have the required permissions to execute the function.
+   */
+  findAll(options?: OptionsWithRql): Promise<Dispatcher[]>;
+
+  /**
+   * ## Retrieve a paged list of Dispatchers
+   *
+   * **Global Permissions:**
+   * - `VIEW_DISPATCHERS` - Allows a user to view Dispatchers
+   *
+   * @param options {@link OptionsWithRql} - Add filters to the requested list
+   * @returns An iterator for the queried providers {@link FindAllIterator FindAllIterator<Dispatcher>}
+   * @throws {@link NoPermissionError} when the user doesn't have the required permissions to execute the function.
```

Comment:
This is also not related to functions and we decided to not add the errors here I think 🤔

Reply:
Same comments in the findAll 😂